pkg/innerring/innerring.go

```diff
@@ -985,15 +983,13 @@ func (s *Server) createClient(ctx context.Context, p chainParams, errChan chan<-
 	if len(endpoints) == 0 {
 		return nil, fmt.Errorf("%s chain client endpoints not provided", p.name)
 	}
-
-	return client.New(
-		p.key,
+	var options = []client.Option{
 		client.WithContext(ctx),
 		client.WithLogger(p.log),
-		client.WithDialTimeout(p.cfg.GetDuration(p.name+".dial_timeout")),
+		client.WithDialTimeout(p.cfg.GetDuration(p.name + ".dial_timeout")),
```

Comment:
go fmt did this? constructor (func args) vs slice creation? interesting

Reply:
Seems to be like fmt, yes.